### DISCUSSION TITLE
Getting Attributes from SQS messages

### DISCUSSION THREAD

**Discussion by johnxy84** [vectordotdev/vector#18577] [Q&A]
Right now I'm working with SQS sink and it all work fine but I only get the body of the SQS message, not the attributes

![image](https://github.com/vectordotdev/vector/assets/16444619/9905f62a-8858-459d-9f41-485b42f6b81e)

This is what the output of the sqs sink right now is
![image](https://github.com/vectordotdev/vector/assets/16444619/7d5198c4-197d-4111-b82e-354f57f4b3d0)

I wonder if it's possible to also get that before we go on ahead to start rewriting things all over

**Comment by jszwedko** [comment]
Hi @johnxy84 ,

I think this would be a feature request if you want to open one.

**Reply by johnxy84** [reply]
Thanks, I created a feature request https://github.com/vectordotdev/vector/issues/18877
